### ISSUE TITLE
CLI command should be changed

### ISSUE THREAD

**Issue by sjimenez77** [devonfw/ide#481]
The CLI command that we use with the devonfw IDE is `devon`. In parallel, we are still struggling to make the devonfw name good spelled in many different places and publications that it still appears as just devon. This is not correct and does not follow the trademark that with much effort we achieved recently. IMHO, by using the command `devon` we are prolonging the situation and the errors.

My proposal is to make the command just different from `devon`, so maybe something like `dfw` could work as something short and recognizable. Also an alias could just work for a soft transition. 

Does it make sense to you? What do you think @hohwille, @maybeec, @ivanderk, @ssarmokadam, @sujith-mn?

**Comment by markusschuh** [comment]
At least you should not start using another trademark (DFW) as cli name, when a potential trademark topic may be the reason to do the change at all ... You potentially won't identify any 3char abbrev without a trademark today.
Also you could instruct the users, how to do the easy changes, when they want to maintain there own alias shortened for whatever shall be the future *ide command. Even CMD has them ( doskey d=devonfwide )

**Comment by sjimenez77** [comment]
Good point. The idea of something like `dfw` was to use some kind of devonfw acronym as Angular does with `ng` for example. 

Some time ago @hohwille and me already discussed this, and we agreed that using `devonfw` as command was a bit long for users. 

In any case, from my point of view `devon` is not the good option as I still see people confusing devonfw name with devon. I do not want that people creates its own aliases either, but creating the best recognizable devonfw CLI command (besides, _devon_ is already a trademark, a place and some other things...).

**Comment by maybeec** [comment]
I understand your point @sjimenez77 although I barely doubt that this "devon" notices are not coming from the devonfw ide.
For me there are two main reasons behind it:
1. even the core team is not consistent in their use of writing devonfw sometimes
2. devonfw is always pronounced "devon" on all oral channels. For me this is the main killer actually and I don't see any option to prevent that for the moment.

**Comment by hohwille** [comment]
A must have for a reasonable CLI is that it is short.
So if we are going to change the name `devon` it should not get longer and more cryptical like `devonfw` but instead be shortened to `dev`. That would even make it more appealing for developers as `dev` is associated with `development` otherwise we could have used `ide` but changing this now is IMHO too late - consider that we already have a commandlet called `ide`.
So I would either close this issue as wontfix of shorten the alias and script name to `dev` (what will cause some small breaking changes and confusion for our users).
